spark/v3.5/spark-extensions/src/test/java/org/apache/iceberg/spark/extensions/TestSetWriteDistributionAndOrdering.java

```diff
@@ -277,7 +302,7 @@ public void testSetWriteDistributedByOnly() {
     Table table = validationCatalog.loadTable(tableIdent);
     assertThat(table.sortOrder().isUnsorted()).as("Table should start unsorted").isTrue();
 
-    sql("ALTER TABLE %s WRITE DISTRIBUTED BY PARTITION UNORDERED", tableName);
```

Comment:
Why does this test need a change?

Reply:
Otherwise, it's a copy-paste of the test `testSetWriteDistributedByAndUnordered` above.